generate_blockips.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euo pipefail
+
+app=${1}
+space=${2}
+org=${3}
+
+# Target space
+cf target -o ${org} -s ${space}
+
+# Generate blockips.conf from CF env var
+echo "Generating blockips.conf from Cloud Foundry environment variable..."
+APP_GUID=$(cf app "$app" --guid)
+BLOCKED_IPS=$(cf curl "/v3/apps/${APP_GUID}/environment_variables" | jq -r '.var.BLOCKED_IPS')
```

Comment:
@tmpayton I've been thinking about this and it will probably be better to use our bound service creds like (fec-creds-*) instead of the direct app env vars like this.  Can you update the script to pull from our bound service instead?